README.rst

```diff
@@ -1,16 +1,24 @@
-.. |I2C| replace:: I\ :sup:`2`\ C
 
-lib_i2c
-#######
+:orphan:
 
-:Version: 6.2.0
-:Vendor: XMOS
+.. |I2C| replace:: I\ :sup:`2`\ C
 
-:Scope: General Use
+#########################
+lib_i2c: XMOS I2C Library
```

Comment:
Missed using the |I2C| replacement here (or just use I²C :-P)

Reply:
done